File: app/src/main/java/guillermo/com/pugme/application/PugsApplication.java
package guillermo.com.pugme.application;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;

/**
 * Created by guillermo.rosales on 21/01/16.
 */
public class PugsApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(getApplicationContext());
    }
}

File: app/src/main/java/guillermo/com/pugme/application/PugsApplication.kt
package guillermo.com.pugme.application

import android.app.Application

import com.facebook.drawee.backends.pipeline.Fresco

/**
 * Created by guillermo.rosales on 21/01/16.
 */
class PugsApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Fresco.initialize(applicationContext)
    }
}
